setup.py

```diff
@@ -42,7 +41,9 @@ def get_requirements():
 
     # NOTE: We use latest version under Python 3
     required = [l for l in required if not l.startswith('cmd2')]
-    if six.PY2:
+    # same as six.PY2, but we can't use six in setup.py  because it isn't
+    # garaunteed to be in a fresh virtualenv before installing this package
```

Comment:
Wow, you are not kidding, that is the same as [`six.PY2`](https://github.com/benjaminp/six/blob/master/six.py#L36):

```python
PY2 = sys.version_info[0] == 2
```